example/webgpu_gpuPathTracingSimpleCompute.js

```diff
@@ -0,0 +1,338 @@
+import * as THREE from "three";
+import { OrbitControls } from "three/addons/controls/OrbitControls.js";
+import { FullScreenQuad } from 'three/addons/postprocessing/Pass.js';
+import Stats from 'three/addons/libs/stats.module.js';
+import { GUI } from 'three/addons/libs/lil-gui.module.min.js';
+import {
+	attribute, uniform, wgslFn, varyingProperty, instanceIndex, textureStore, texture,
+	storage, cameraProjectionMatrix, modelWorldMatrix, cameraViewMatrix
+} from "three/tsl";
+
+
+import {
+	MeshBVH,
+	MeshBVHBufferArrays,
+	SAH
+} from '../src/index.js';
+
+import { intersectsBVHNodeBounds, intersectsBounds, ndcToCameraRay, normalSampleBarycoord } from '../src/gpu/wgsl/common_functions.wgsl.js';
+import { intersectsTriangle, intersectTriangles, bvhIntersectFirstHit } from '../src/gpu/wgsl/bvh_ray_functions.wgsl.js';
```

Comment:
Thanks! This is great -

It looks like this PR doesn't include these files -- can we adjust it so it's complete and we can work from this one?